### DISCUSSION TITLE
Replication among multiple slave nodes of redis

### DISCUSSION THREAD

**Discussion by aadhik** [longhorn/longhorn#4168] [Q&A]
I have a redis master and 3 slave setup , currently using the native redis replication for HA .
Is it possible using  longhorn , i run 3 instances of redis as stateful set , and replication among the three instaces is taken care by longhorn . I understand , using replication of longhorn , i can achieve HA just by running single instances of redis , but want to explore the above setup , if possible could extend it to mongodb as well

**Comment by derekbit** [comment]
You probably can take a look RWX volume and check if the 3 instances can use the same RWX volume.
https://longhorn.io/docs/1.3.0/advanced-resources/rwx-workloads/
